### PR TITLE
Add support for Apollo87HL and Apollo87HL-T

### PR DESCRIPTION
**Pull request by Gondolindrim**
## Description

Adds support for Apollo87HL and Apollo87HL-T.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/acheron/apollo/87hlt/keyboard.json [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        "pin": "B15"
+    },
+    "rgblight": {
+        "led_count": 3,
+        "driver": "ws2812",
+        "sleep": true
+    },
+    "diode_direction": "COL2ROW",
+    "layouts": {
+    "LAYOUT": {
```
Comment (dunk2k) [inline keyboards/acheron/apollo/87hlt/keyboard.json:61]:
```suggestion
    "community_layouts": ["tkl_ansi_tsangan_split_bs_rshift", "tkl_ansi_wkl_split_bs_rshift"],
    "layouts": {
        "LAYOUT_tkl_ansi_tsangan_split_bs_rshift": {
```

File: keyboards/acheron/apollo/87hl/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, MO(1)  , KC_RCTL,    KC_LEFT, KC_DOWN, KC_RIGHT
+        ),
+
+   [1] = LAYOUT(
+        QK_BOOT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                              
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,             KC_TRNS,   
+        KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS
```
Comment (dunk2k) [inline keyboards/acheron/apollo/87hl/keymaps/default/keymap.c:35]:
```suggestion
   [0] = LAYOUT_tkl_ansi_split_bs_rshift(
        KC_ESC ,          KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,             KC_PSCR, KC_SCRL, KC_PAUS, 
        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_BSPC,    KC_INS , KC_HOME, KC_PGUP,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS,             KC_DEL , KC_END , KC_PGDN,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,          KC_ENT ,                                   
        KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, MO(1)  ,                      KC_UP,            
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, MO(1)  , KC_RCTL,             KC_LEFT, KC_DOWN, KC_RIGHT
        ),

   [1] = LAYOUT_tkl_ansi_split_bs_rshift(
        QK_BOOT,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______, 
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                              
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,   
        _______, _______, _______,                            _______,                            _______, _______, _______, _______,             _______, _______, _______
```

File: keyboards/acheron/apollo/87hl/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, MO(1)  , KC_RCTL,    KC_LEFT, KC_DOWN, KC_RIGHT
+        ),
+   [1] = LAYOUT(
+        QK_BOOT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                              
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,             KC_TRNS,   
+        KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS
```
Comment (dunk2k) [inline keyboards/acheron/apollo/87hl/keymaps/via/keymap.c:34]:
```suggestion
   [0] = LAYOUT_tkl_ansi_split_bs_rshift(
        KC_ESC ,          KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,             KC_PSCR, KC_SCRL, KC_PAUS, 
        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_BSPC,    KC_INS , KC_HOME, KC_PGUP,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS,             KC_DEL , KC_END , KC_PGDN,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,          KC_ENT ,                                   
        KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, MO(1)  ,                      KC_UP,            
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                            KC_RALT, KC_RGUI, MO(1)  , KC_RCTL,             KC_LEFT, KC_DOWN, KC_RIGHT
        ),

   [1] = LAYOUT_tkl_ansi_split_bs_rshift(
        QK_BOOT,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______, 
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                              
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,   
        _______, _______, _______,                            _______,                            _______, _______, _______, _______,             _______, _______, _______
```

File: keyboards/acheron/apollo/87hlt/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                                     KC_RALT, KC_RGUI, KC_RCTL,    KC_LEFT, KC_DOWN, KC_RIGHT
+        ),
+   [1] = LAYOUT(
+        QK_BOOT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                              
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,             KC_TRNS,   
+        KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                                     KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS
```
Comment (dunk2k) [inline keyboards/acheron/apollo/87hlt/keymaps/default/keymap.c:34]:
```suggestion
   [0] = LAYOUT_tkl_ansi_tsangan_split_bs_rshift(
        KC_ESC ,          KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,             KC_PSCR, KC_SCRL, KC_PAUS, 
        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_BSPC,    KC_INS , KC_HOME, KC_PGUP ,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS,             KC_DEL , KC_END , KC_PGDN ,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,          KC_ENT ,                                   
        KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, MO(1)  ,                      KC_UP,            
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                                     KC_RALT, KC_RGUI, KC_RCTL,             KC_LEFT, KC_DOWN, KC_RIGHT
        ),
   [1] = LAYOUT_tkl_ansi_tsangan_split_bs_rshift(
        QK_BOOT,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______, 
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                              
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,   
        _______, _______, _______,                            _______,                                     _______, _______, _______,             _______, _______, _______
```

File: keyboards/acheron/apollo/87hlt/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                                     KC_RALT, KC_RGUI, KC_RCTL,    KC_LEFT, KC_DOWN, KC_RIGHT
+        ),
+   [1] = LAYOUT(
+        QK_BOOT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS,
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS, 
+        KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,                              
+        KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,             KC_TRNS,   
+        KC_TRNS, KC_TRNS, KC_TRNS,                            KC_TRNS,                                     KC_TRNS, KC_TRNS, KC_TRNS,    KC_TRNS, KC_TRNS, KC_TRNS
```
Comment (dunk2k) [inline keyboards/acheron/apollo/87hlt/keymaps/via/keymap.c:34]:
```suggestion
   [0] = LAYOUT_tkl_ansi_tsangan_split_bs_rshift(
        KC_ESC ,          KC_F1  , KC_F2  , KC_F3  , KC_F4  , KC_F5  , KC_F6  , KC_F7  , KC_F8  , KC_F9  , KC_F10 , KC_F11 , KC_F12 ,             KC_PSCR, KC_SCRL, KC_PAUS, 
        KC_GRV , KC_1   , KC_2   , KC_3   , KC_4   , KC_5   , KC_6   , KC_7   , KC_8   , KC_9   , KC_0   , KC_MINS, KC_EQL , KC_BSPC, KC_BSPC,    KC_INS , KC_HOME, KC_PGUP ,
        KC_TAB , KC_Q   , KC_W   , KC_E   , KC_R   , KC_T   , KC_Y   , KC_U   , KC_I   , KC_O   , KC_P   , KC_LBRC, KC_RBRC, KC_BSLS,             KC_DEL , KC_END , KC_PGDN ,
        KC_CAPS, KC_A   , KC_S   , KC_D   , KC_F   , KC_G   , KC_H   , KC_J   , KC_K   , KC_L   , KC_SCLN, KC_QUOT,          KC_ENT ,                                   
        KC_LSFT,          KC_Z   , KC_X   , KC_C   , KC_V   , KC_B   , KC_N   , KC_M   , KC_COMM, KC_DOT , KC_SLSH, KC_RSFT, MO(1)  ,                      KC_UP,            
        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC ,                                     KC_RALT, KC_RGUI, KC_RCTL,             KC_LEFT, KC_DOWN, KC_RIGHT
        ),
   [1] = LAYOUT_tkl_ansi_tsangan_split_bs_rshift(
        QK_BOOT,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______,
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,             _______, _______, _______, 
        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,                              
        _______,          _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,                      _______,   
        _______, _______, _______,                            _______,                                     _______, _______, _______,             _______, _______, _______
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
add Community Layout Support

File: keyboards/acheron/apollo/87hlarc1/keyboard.json
Comment (fauxpark) [inline keyboards/acheron/apollo/87hlarc1/keyboard.json:1]: Please run `qmk format-json -i` on this file.

File: keyboards/acheron/apollo/87hlarc2/keyboard.json
Comment (fauxpark) [inline keyboards/acheron/apollo/87hlarc2/keyboard.json:1]: Please run `qmk format-json -i` on this file.

File: keyboards/acheron/apollo/87hltarc1/keyboard.json
Comment (fauxpark) [inline keyboards/acheron/apollo/87hltarc1/keyboard.json:1]: Please run `qmk format-json -i` on this file.

File: keyboards/acheron/apollo/87hltarc2/keyboard.json
Comment (fauxpark) [inline keyboards/acheron/apollo/87hltarc2/keyboard.json:1]: Please run `qmk format-json -i` on this file.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/acheron/apollo/87hlarc1/87hlarc1.c [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+void keyboard_post_init_kb() {
+    led_update_kb((led_t)host_keyboard_leds());
+}
```
Comment (drashna) [inline keyboards/acheron/apollo/87hlarc1/87hlarc1.c:21]:
```suggestion
    keyboard_post_init_user();
}
```

File: keyboards/acheron/apollo/87hlarc1/keyboard.json [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "rgblight": true
+    },
```
Comment (drashna) [inline keyboards/acheron/apollo/87hlarc1/keyboard.json:28]:
```suggestion
        "bootmagic": true,
        "extrakey": true,
        "nkro": true,
        "rgblight": true
    },
```

File: keyboards/acheron/apollo/87hlarc2/87hlarc2.c [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+void keyboard_post_init_kb() {
+    led_update_kb((led_t)host_keyboard_leds());
+}
```
Comment (drashna) [inline keyboards/acheron/apollo/87hlarc2/87hlarc2.c:21]:
```suggestion
    keyboard_post_init_user();
}
```

File: keyboards/acheron/apollo/87hltarc1/87hltarc1.c [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+void keyboard_post_init_kb() {
+    led_update_kb((led_t)host_keyboard_leds());
+}
```
Comment (drashna) [inline keyboards/acheron/apollo/87hltarc1/87hltarc1.c:21]:
```suggestion
    keyboard_post_init_user();
}
```

File: keyboards/acheron/apollo/87hltarc2/87hltarc2.c [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+void keyboard_post_init_kb() {
+    led_update_kb((led_t)host_keyboard_leds());
+}
```
Comment (drashna) [inline keyboards/acheron/apollo/87hltarc2/87hltarc2.c:21]:
```suggestion
    keyboard_post_init_user();
}
```

**Review by drashna — CHANGES_REQUESTED** [review]
As of August 26, 2024, `qmk/qmk_firmware` is no longer accepting VIA-enabled keymaps as these have now transitioned to a repository under the VIA team's control.

As you've submitted a PR containing `via` or VIA-enabled keymap(s), this is your notice that they should be removed from this PR. You should now submit a secondary PR to the [VIA QMK Userspace](https://github.com/the-via/qmk_userspace_via) repository with your associated `via` or VIA-enabled keymaps instead.

**Pull request closed without merging**